fossasia/static/custom.css

```diff
@@ -176,3 +177,10 @@ footer.classic .social-profiles {
 .menu li a:hover {
   border-bottom: 2px solid #fff;
 }
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  /* Set the fixed height of the footer here */
+  height: 60px;
```

Comment:
Your footer height seems to be around 185px; for it to work properly on all screens.